kafka_topic_enforcer/src/main/java/com/tesla/data/topic/enforcer/StrimziCommand.java

```diff
@@ -0,0 +1,95 @@
+/*
+ * Copyright (C) 2024 Tesla Motors, Inc. All rights reserved.
+ */
+
+package com.tesla.data.topic.enforcer;
+
+import com.tesla.data.enforcer.BaseCommand;
+
+import com.beust.jcommander.Parameter;
+import com.beust.jcommander.Parameters;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.dataformat.yaml.YAMLFactory;
+import com.fasterxml.jackson.dataformat.yaml.YAMLGenerator;
+import com.google.common.base.CaseFormat;
+import com.google.common.hash.Hashing;
+import io.strimzi.api.kafka.model.topic.KafkaTopic;
+import io.strimzi.api.kafka.model.topic.KafkaTopicBuilder;
+
+import java.nio.charset.StandardCharsets;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+@Parameters(commandDescription = "Generate the Strimzi KafkaTopic resources YAML on stdout from the topic config. " +
+    "Certain information such as topic tags and config comments would be missed.")
+public class StrimziCommand extends BaseCommand<ConfiguredTopic> {
+
+  @Parameter(
+      names = {"--kafka_name", "-k"},
```

Comment:
`kafka_name` sounds pretty confusing - is this a cluster name or a topic name or some other name? How about making this explicit, like `strimzi-kafka-cluster-name`?

Reply:
Since this command is a "Strimzi" command, the word strimzi is redundant in the context. I'm switching to `kafka_cluster` instead then.